pages/talks/lunch-chats-mon-1.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: talk
+title: Sprint preparation, Git and PRS
+description: 
+---
+
+During this year you will learn about git and PRs.
```

Comment:
should `year` not be something more specific like `session`?

Reply:
thank you matthijs